guides/8-perpetual-exchange/2-how-it-works.md

```diff
@@ -172,6 +208,25 @@ There are 4 types of fees on Jupiter Perpetual:
 
 A flat rate of **0.06%** of the position amount is charged when opening or closing a position. This base fee is also charged when a position is closed partially.
 
+To calculate the base open or close for a trade:
+
+```
+BPS_POWER = 10^4      // 10_000
+
+Calculate Price Impact Fee:
```

Comment:
this is base fee

Reply:
Oh yah that is wrong! Made the fix here: https://github.com/jup-ag/space-station/pull/539/commits/4092dd2d819885a3373efa47eb64128b6c5eafd4

Will work with Miir to fix the docs formatting and language later